### PR TITLE
Add additional options to texture converter

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: properties/scene.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ class AppleseedTextureConvertProps(bpy.types.PropertyGroup):
                                               ('linear', "Linear", ""),
                                               ('sRGB', "sRGB", ""),
                                               ('Rec709', "Rec.709", "")],
-                                         default='sRGB')
+                                         default='linear')
```
Comment (dictoon) [inline properties/scene.py]: Is that change intended? If so, it probably deserves a separate commit.

File: projectwriter.py [hunk 2]
```diff
@@ -801,8 +802,14 @@ def __get_osl_node_params(self, node, scene):
                     parameter = bpy.path.abspath(parameter)
                     if scene.appleseed.sub_textures is True:
                         if parameter.endswith(image_extensions):
-                            base_filename = parameter.split('.')
-                            parameter = "{0}.tx".format(base_filename[0])
+                            image_path_segments = os.path.split(parameter)
```
Comment (dictoon) [inline projectwriter.py]: What's `image_path_segments` exactly? You seem to assume that `image_path_segments[0]` is a folder name and `image_path_segments[1]` is a file name?

File: properties/scene.py [hunk 3]
```diff
@@ -88,6 +88,15 @@ def update_stamp(self, context):
 
     # Texture conversion
 
+    tex_output_dir = bpy.props.StringProperty(name="tex_output_dir",
+                                              description="",
+                                              default="",
+                                              subtype='DIR_PATH')
+
+    tex_output_use_cust_dir = bpy.props.BoolProperty(name="tex_output_use_cust_dir",
+                                                         description="",
```
Comment (dictoon) [inline properties/scene.py]: Fix indentation.

File: projectwriter.py (hunk 2 above)
Comment (jdent02) [reply projectwriter.py]: It’s the best way I could come up with of saving information about the original texture directory in case a custom one is not used

File: properties/scene.py (hunk 1 above)
Comment (jdent02) [reply properties/scene.py]: Yes. The majority of textures in a physically based material are linear and it was a pain in the ass having to switch so many textures from the old sRGB default

Comment (dictoon) [reply properties/scene.py]: Ok. Ideally we would guess the color space from the file extension (HDR file formats such as OpenEXR, HDR, floating point TIFF -> linear; everything else -> sRGB).

Also please make a separate commit for this.

Comment (jdent02) [reply properties/scene.py]: The guessing wouldn't work for something like Substance Designer.  It exports everything at the same bit depth.

You should get them to change that :)

Comment (jdent02) [reply properties/scene.py]: This part is in a separate commit

File: projectwriter.py (hunk 2 above)
Comment (jdent02) [reply projectwriter.py]: I added comments explaining why those steps are being taken.

Comment (dictoon) [reply projectwriter.py]: > I added comments explaining why those steps are being taken.

In the code? I'm not seeing them.

**Comment by jdent02** [comment]
Okay now it should all be up to date.
